frontend/src/features/properties/selector/search/PropertySearchSelectorFormView.tsx

```diff
@@ -33,7 +33,7 @@ export const PropertySearchSelectorFormView: React.FunctionComponent<IPropertySe
 }) => {
   return (
     <>
-      <StyledFormSection>
+      <Section header={undefined}>
         <Styled.H3>Search for a property</Styled.H3>
```

Comment:
We should figure out if we want different types of section headers and standardize those so we can re-use the component instead of just using the style

Reply:
Agreed